src/Microsoft.VisualStudio.SDK.Analyzers.Tests/DiagnosticVerifier.cs

```diff
@@ -169,6 +169,7 @@ protected static Project CreateProject(string[] sources, string language = Langu
                 Environment.GetEnvironmentVariable("USERPROFILE"),
                 ".nuget",
                 "packages");
+            nugetPackageRoot = @"e:\nuget\packages";
```

Comment:
This of course needs to be reverted.

Reply:
Correct, I didn't intend to push this change. Will revert